projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-ai-suggestions-from-ai/index.ts

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { useCallback, useEffect, useRef } from '@wordpress/element';
+/**
+ * Types
+ */
+import { SuggestionsEventSource, askQuestion } from '../../lib/suggestions';
+import type { PromptItemProps } from '../../lib/prompt';
+
+type useSuggestionsFromAIOptions = {
```

Comment:
I thinks it's neater to have types with UpperCamelCase:

```suggestion
type UseSuggestionsFromAIOptions = {
```